src/backend/distributed/shardsplit/shardsplit_decoder.c

```diff
@@ -73,10 +277,20 @@ static void
 split_change_cb(LogicalDecodingContext *ctx, ReorderBufferTXN *txn,
```

Comment:
name should perhaps be updated to reflect logic changes

Reply:
OK renamed it to "shard_split_and_cdc_change_cb"